src/Component/ColorPicker/index.js

```diff
@@ -10,56 +10,54 @@ import PrimaryButton from 'Component/Button/Primary';
 import SecondaryButton from 'Component/Button/Secondary';
 import SimpleInput from 'Component/Input/Simple';
 import Toggle from 'Component/Toggle';
+import {formatColorAsRgbOrHex, getHexColorOrNull, getRgbaColorOrNull, isRgb, rgbFormat} from 'Util/colorHexRgbHelper';
```

Comment:
CONSIDER:

1. Move helpers to `src/Component/ColorPicker/Util`. As we are not likely to use them anywhere else.

You can rename `formatColorAsRgbOrHex` to just `formatColor` after that.

2. Use a separate file for each helper.